src/Middleware/SwaggerJson.php

```diff
@@ -8,13 +8,12 @@
 use OpenApi\Annotations\OpenApi;
 use Psr\Http\Message\ResponseInterface;
 use Psr\Http\Message\ServerRequestInterface;
-use Psr\Http\Server\MiddlewareInterface;
 use Psr\Http\Server\RequestHandlerInterface;
 use Yiisoft\Cache\CacheInterface;
 use Yiisoft\DataResponse\DataResponseFactoryInterface;
 use Yiisoft\Swagger\Service\SwaggerService;
 
-final class SwaggerJson implements MiddlewareInterface
+final class SwaggerJson implements RequestHandlerInterface
```

Comment:
1. What's the benefit?
2. It's still in the `Middleware` namespace.

Reply:
I'll move it outside of the `middleware` folder